docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,256 @@
+
+
+The `node details` (`Node ID`, `Network Address`, `Networking Key` and `Staking Key`) that need to be submitted when staking the node on Flow Port, can be found in the file: `./bootstrap/public-root-information/node-info.pub.<node-id>.json`.
+
+```shell:title=Example
+$cat ./bootstrap/public-root-information/node-info.pub. e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5.json
+{
+  "Role": "access",
+  "Address": "flowaccess.mycompany.com:3569",
+  "NodeID": "e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5",
+  "Weight": 1000,
+  "NetworkPubKey": "f493a74704f6961ae7903e062ecd58d990672858eff99aece7bfbccf3aa02de8f1a624ecbf21a01e8b2f4a5854c231fbe218edd7762a34fea881f3958a215305",
+  "StakingPubKey": "ae8dcf81f3a70d72036b7ba2c586ed37ed0eb82b9c0a4aab998a8420f98894f94c14f84fa716e93654d3940fc0c8ff4d19b504c90a5b4918b28f421e9d3659dc2b7e246025ebeffea0d83cceefe315d7ed346dbe412fdac51b64997d97d29f7e"
+}
+```
+
+### Timing
+
+New nodes are able to join the network each time a new epoch begins.
+An epoch is a period of time (approximately one week) when the node operators in the network are constant.
+At epoch boundaries, newly staked node operators are able to join the network and existing node operators which have unstaked may exit the network.
+You can read more about epochs [here](https://developers.flow.com/nodes/staking/schedule).
+
+
+In order to join the network at epoch N+1, the node **must** be registered with at least 100 FLOW staked prior to the end of epoch N's Staking Auction Phase.
+Currently, an epoch transition on mainnet starts on Wednesday (Pacific time) early morning. Hence, the node should be staked by EOD Tuesday (Pacific time) to be able to join the network starting on Wednesday.
+
+Confirmation of a new node's inclusion in epoch N+1 is included in the [`EpochSetup` event](/staking/epoch-scripts-events/#epochsetup).
+
+![Flow Epoch Schedule](https://storage.googleapis.com/flow-resources/documentation-assets/epoch-startup-order.png)
+
+
+## Step 3 - Verify if your node ID was selected
+
+There are two main ways to verify if your node was selected. If you can monitor events, look for [the epoch setup event](/staking/epoch-scripts-events/#epochsetup) that gets emitted by the epoch contract. That event is emitted at the end of epoch N's staking auction and contains a list of node IDs that are confirmed for the next epoch.
+
+Another way to verify is the query the information directly. After the staking auction ends, you can use the [get node info script](/staking/epoch-scripts-events#get-all-of-the-info-associated-with-a-single-node-staker) to get the information about your node. If your tokens are shown in the `tokensCommitted` or `tokensStaked` balance, you are confirmed for the next epoch. If the tokens are in the `tokensUnstaked` balance, then your node was not chosen.
```

Comment:
@jordanschalm and @joshuahannan - can we use the Flow CLI here instead and say something like this?

Check your current staking-info. It it says token Committed then your node has been selected.

> 
> $ flow accounts staking-info 0x32a1b5d0a0bc7f33  -n testnet
> 
> Account staking info:
> 	ID: 						 "5f6c73a22445d7d958c6a37c1f3be99c72cacd39894a3e46d6647a9adb007b4d"
> 	Initial Weight: 				 100
> 	Networking Address: 				 "execution-001.devnet41.nodes.onflow.org:3569"
> 	Networking Key: 				 "5db0d4a2aa1f72d6bd03decfddb3a45199a8f42321dce09a176aed85f8c76e82a969a721968e99dca3204a311b7d681a12b5e97450dfb15ba9f8bfabbd91ce80"
> 	Role: 						 3
> 	Staking Key: 					 "8c30fdf402de3861d2c78189e16e46a9d79ac6eebdfc0c54a92e283999997fb568c65843127dbec0c8b80d153d3167d616b1590844a4a6d678b40335938ded790778a868809f215f521256a422021ab411616ccba03c5a6dfcdf0beb35cc114d"
> 	Tokens Committed: 				 0.00000000
> 	Tokens To Unstake: 				 0.00000000
> 	Tokens Rewarded: 				 7413960.66250000
> 	Tokens Staked: 					 1250000.00000000
> 	Tokens Unstaked: 				 0.00000000
> 	Tokens Unstaking: 				 0.00000000
> 	Node Total Stake (including delegators): 	 1250000.00000000
> 
> Account has no delegations.

Reply:
I can add examples for before and after, but this should be a way for operators to check if they got selected or not, correct?

Reply:
updated the doc with cli and flowport example on how to check if node got selected.